Samples/EFCodeFirst/EntityFrameworkCodeFirst.csproj

```diff
@@ -1,20 +1,20 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;net8.0</TargetFrameworks>
     <OutputType>Exe</OutputType>
   </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), Build.props))\Build.props" />
   <ItemGroup>
-    <PackageReference Include="EnterpriseLibrary.TransientFaultHandling.Data.NetCore" Version="6.0.1312" />
-    <PackageReference Include="EnterpriseLibrary.TransientFaultHandling.NetCore" Version="6.0.1312" />
-    <PackageReference Include="Microsoft.Azure.SqlDatabase.ElasticScale.Client" Version="2.3.0" />
-    <PackageReference Include="System.Configuration.ConfigurationManager" Version="5.0.0" />
     <PackageReference Include="EntityFramework" Version="6.4.4" />
+    <PackageReference Include="System.Data.SqlClient" Version="4.8.5" />

```

Comment:
I see System.Data.SqlClient referenced here, but throughout the code I see Microsoft.Data.SqlClient.  Is that correct?

Furthermore, I see Release 2.4.0 still is referencing Microsoft.Data.SqlClient 3.0.0 which is not the latest package which is 5.1.2.  I believe the higher version package for Microsoft.Data.SqlClient will include a higher version of Azure.Identity.

https://www.nuget.org/packages/Microsoft.Data.SqlClient

Reply:
@tps-pcaldwell are you looking to use EF (Classic).   The EF (Classic) sample hasn't been updated, because there has been no demand for it to be updated.  Are you looking to use EF (Classic), or are you looking to use EF Core?

Reply:
Yes, I'm looking to use EF Core.  In addition, the sample code is using the nuget reference, so it is still reference 2.3.0 instead of a project reference.  I would recommend updating the nuget reference to 2.4.0 or removing the EF sample code which would be a shame.

Reply:
Yeah, I don't think we can update the EF Classic sample to 2.4 (which is .Net Core).  We'll need to add another sample alongside the EF Classic sample for EF Core, which would use 2.4.  

Are you interested in an EF Core sample using 2.4?